packages/containerd/containerd-cri-base-json

```diff
@@ -102,6 +102,15 @@ oci-defaults = { version = "v1", helpers = ["oci_defaults"] }
                 "mode=755",
                 "size=65536k"
             ]
+        },
+        {
+            "destination": "/sbin/modprobe",
```

Comment:
One risk of this implementation is that if `PATH` is `/usr/bin:/sbin`, the `modprobe` binary in the container will be used instead of the injected `modprobe`. I guess that's a calculated risk to take, since we can't inject `modprobe` in every single path in `PATH`. Probably something to follow up for a later release is checking `PATH` and inject `modprobe` to the first path in that list.